src/examples/rtc/rtc.go

```diff
@@ -0,0 +1,61 @@
+//go:build rp2040
+
+package main
+
+// This example demonstrates real time clock (RTC) support.
+//
+// An alarm can be set to execute user callback function once or on a schedule.
+//
+// Unfortunately, it is not possible to use time.Time to work with RTC,
+// that would introduce a circular dependency between "machine" and "time" packages.
+
+import (
+	"fmt"
+	"machine"
+	"time"
+)
+
+// For RTC to work, it must be set to some reference time first
+var rtcTimeReference = machine.RtcTime{
+	Year:  2023,
+	Month: 01,
+	Day:   19,
+	Dotw:  4,
+	Hour:  1,
+	Min:   12,
+	Sec:   45,
+}
+
+// Alarm shall fire every minute at 5 sec
+var rtcTimeAlarm = machine.RtcTime{
+	Year:  -1,
+	Month: -1,
+	Day:   -1,
+	Dotw:  -1,
+	Hour:  -1,
+	Min:   -1,
+	Sec:   5,
+}
+
+func main() {
```

Comment:
This RTC does not keep state on power down or reboot, unfortunately.
So one has to source the correct time somewhere first, like from WiFi Access Point, as I do in some of my projects.

When one just wants to schedule a delayed interrupt (say within minutes, hours), the reference time does not really matter. It can be 1970-01-01, really.

Reply:
Implemented in example. Thanks for the idea!